builtin/mainmenu/init.lua

```diff
@@ -78,7 +78,31 @@ local function init_globals()
 		end,
 		-- Filter function
 		function(element, gameid)
-			return element.gameid == gameid
+			if element.gameid == gameid then
+				return true
+			end
+			local gameconfig = Settings(pkgmgr.find_by_gameid(gameid).path .. "/game.conf")
+			local aliases = (gameconfig:get("gameid_alias") or ""):split()
```

Comment:
I think it would be good to make the `","` as a delimiter explicit here. I'd probably prefer Lua-style "splitting" via `for alias in gameconfig:get("gameid_alias"):gmatch("[^,]+") do`.